apps/style/applab/style.scss

```diff
@@ -515,3 +515,48 @@ button.arrow:focus {
 .divApplabDesignMode canvas {
   background-color: #eee;
 }
+
+#headers #design-toolbox-header {
+  width: 270px;
+  border-right: 1px solid gray;
+  float: left;
+}
+
+#designModeBox {
+  position:absolute;
+  top: 40px;
+  bottom: 0px;
+  width: 100%;
+}
+
+#design-mode-container {
+  position:absolute;
+  width: 100%;
+  top: 0px;
+  bottom: 0px;
+  background-color: white;
+  box-sizing: border-box;
+  border-left: 1px solid gray;
+  border-right: 1px solid gray;
+  border-bottom: 1px solid gray;
+}
+
+#design-elements {
+  position:absolute;
+  top: 0px;
+  bottom: 0px;
+  width: 270px;
+  box-sizing: border-box;
+  border-right: 1px solid gray;
+  padding: 10px;
+}
+
+#design-properties {
+  position:absolute;
+  top: 0px;
+  bottom: 0px;
+  left: 270px;
+  right: 0px;
+  box-sizing: border-box;
+  padding: 10px;
+}
```

Comment:
Let's do inline styles, and see how deep the rabbit hole goes ;-) This slide deck has me never wanting to use css files again: https://speakerdeck.com/vjeux/react-css-in-js 

One notable insight from that deck is that because React styles are javascript objects rather than strings, they can more easily be shared, reused, overridden, etc.

One notable drawback is that designers ask things like "can you show me the stylesheet so I can just change it myself / tell you exactly what to change". However, I think it's reasonable to expect a designer to use Chrome inspector to see what the styles are, and if we package our styles nicely into our React component's props, we could even have designers tweak the styles using the React developer tools:

![screen shot 2015-05-28 at 10 45 22 am](https://cloud.githubusercontent.com/assets/8001765/7866880/315cf9d8-0527-11e5-9215-52f6cbdd6dcf.png)